src/Compilers/CSharp/Portable/CodeGen/EmitExpression.cs

```diff
@@ -1755,6 +1763,14 @@ private void EmitInstanceCallExpression(BoundCall call, UseKind useKind)
                     }
 
                     emitArgumentsAndCallEpilogue(call, callKind, receiverUseKind);
+
+                    _builder.LocalSlotManager.EndScopeOfTrackingAddressedLocals(countBefore, MightEscapeTemporaryRefs(
+                        call,
+                        used: true,
```

Comment:
>used: true [](http://example.com/codeflow?start=24&length=10)

Is there a relatively simple test we could add to test this change?

Reply:
Actually yes, after giving it another thought I came up with a test hitting this. Thanks.